provider/azure/vendor/vendor.json

```diff
@@ -2,24 +2,140 @@
-		{"path":"github.com/Azure/go-autorest/autorest/adal","checksumSHA1":"+sZIvmeyhDh28yRhsXVt9+I7Dhk=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/Azure/go-autorest/autorest/azure","checksumSHA1":"sV4wrmfmuKvYpPRPkoPObeBC77g=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/Azure/go-autorest/autorest/azure/auth","checksumSHA1":"S5giB0ep3QAQtB4cQJlLiaPmcFA=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/Azure/go-autorest/autorest/azure/cli","checksumSHA1":"thSHLNGJe/E+o7BjGctLPb0pzK8=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/Azure/go-autorest/autorest/date","checksumSHA1":"I9jacF5Ym6s2GU8dpi9yUT6I1hM=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/Azure/go-autorest/autorest/to","checksumSHA1":"o3pFknhovwgqplUNXkr6mOpEw24=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/Azure/go-autorest/autorest/validation","checksumSHA1":"uh3gcvSj8u8aPYC+x4YU19Gu8Tw=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/Azure/go-autorest/logger","checksumSHA1":"nXtlS0uRMjqa8+dmeEqo3j3AXEo=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/Azure/go-autorest/tracing","checksumSHA1":"StsBozgf+C8kMv6htCiFBJbmmUw=","revision":"3492b2aff5036c67228ab3c7dba3577c871db200","revisionTime":"2019-10-28T18:08:45Z"},
-		{"path":"github.com/dgrijalva/jwt-go","checksumSHA1":"+TKtBzv23ywvmmqRiGEjUba4YmI=","revision":"dbeaa9332f19a944acb5736b4456cfcc02140e29","revisionTime":"2017-10-19T21:57:19Z"},
-		{"path":"github.com/dimchansky/utfbom","checksumSHA1":"L7NqEfBxi5sjuvjE8hnDimfGemA=","revision":"d2133a1ce379ef6fa992b0514a77146c60db9d1c","revisionTime":"2018-12-05T23:29:56Z"},
-		{"path":"github.com/hashicorp/go-discover/provider/azure","checksumSHA1":"F8qQlAbbXZWigahcusD+1FOgXuU=","revision":"c7c0eb3fc6b6f8024c39cb24f3fe254a72595dd4","revisionTime":"2018-04-27T00:43:23Z"},
-		{"path":"github.com/mitchellh/go-homedir","checksumSHA1":"7zLQC+jG19ndjH24FVh/+ZliHac=","revision":"af06845cf3004701891bf4fdb884bfe4920b3727","revisionTime":"2019-01-27T04:21:35Z"},
-		{"path":"golang.org/x/crypto/pkcs12","checksumSHA1":"8ghRWpo8K3eh920FNT0h/5FtPI4=","revision":"86a70503ff7e82ffc18c7b0de83db35da4791e6a","revisionTime":"2019-11-20T00:30:15Z"},
-		{"path":"golang.org/x/crypto/pkcs12/internal/rc2","checksumSHA1":"p0GC51McIdA7JygoP223twJ1s0E=","revision":"86a70503ff7e82ffc18c7b0de83db35da4791e6a","revisionTime":"2019-11-20T00:30:15Z"}
+		{
```

Comment:
Can you run [`vendorfmt`](https://github.com/magiconair/vendorfmt) on this file?

Reply:
Done